shared/wallets/transaction/index.tsx

```diff
@@ -127,6 +129,33 @@ const Detail = (props: DetailProps) => {
   const textSentenceEnd = props.detailView && props.pending ? '\u2026' : '.'
 
   if (props.isAdvanced) {
+    if (props.trustline) {
+      const assetCode = props.trustline.asset.code
+      const assetIssuer = props.trustline.asset.verifiedDomain || 'Unknown'
+      const asset = (
+        <Text type={'BodySmall'}>
+          <Text type="BodySmallBold">{assetCode}</Text>/{assetIssuer}
+        </Text>
+      )
+      if (props.trustline.remove) {
+        return (
+          <Text type="BodySmall" style={{...{wordBreak: 'break-word'}, ...textStyle}}>
+            You removed a trustline:
+            <Text type="BodySmall" style={textStyle}> </Text>
```

Comment:
Linter cries out ```error  Empty components are self-closing  react/self-closing-comp```.
```<Text type="BodySmall" style={textStyle} />``` doesn't work since there's no space.

What should it be? There should be this space after the colon.
![image](https://user-images.githubusercontent.com/705646/60049712-ddb1ea80-969c-11e9-9942-cac38fa81423.png)

Reply:
You can use `{' '}` to insert a space

Reply:
Thanks!